NICE.Identity.Management/ClientApp/src/views/User/User.tsx

```diff
@@ -1,20 +1,95 @@
-import React from "react";
+import React, { Component } from "react";
+
 import { RouteComponentProps, Link } from "react-router-dom";
+import { Breadcrumbs, Breadcrumb } from "@nice-digital/nds-breadcrumbs";
+import { Panel } from "@nice-digital/nds-panel";
+import { Tag } from "@nice-digital/nds-tag";
+
+import { UserType } from "../../models/types";
+import { Endpoints } from "../../data/endpoints";
+import { UnlockUser } from "../../components/UnlockUser/UnlockUser";
 
 type TParams = { id: string };
 
 type UserProps = {} & RouteComponentProps<TParams>;
 
-export const User = (props: UserProps) => {
-	const { match } = props;
-	const { params } = match;
-
-	return (
-		<div className="grid">
-			<div data-g="12">
-				<p>User: {params.id}</p>
-				<Link to={`/users/${params.id}/roles`}>Roles</Link>
-			</div>
-		</div>
-	);
+type UserState = {
+	data: Array<UserType>;
+	error: string;
 };
+
+export class User extends Component<UserProps, UserState> {
+	constructor(props: UserProps) {
+		super(props);
+
+		this.state = {
+			data: [],
+			error: "",
+		};
+	}
+
+	updateData = (updatedData: Array<UserType>) => {
+		this.setState({ data: updatedData });
+	};
+
+	fetchData = async (url: string) => {
+		try {
+			const response = await fetch(url);
+			const data = await response.json();
+			this.setState({ data });
+		} catch (error) {
+			this.setState({ error });
+		}
+	};
+
+	componentDidMount() {
+		this.fetchData(Endpoints.user(this.props.match.params.id));
+	}
+
+	render() {
+		const { data, error } = this.state;
+		const userDetails = data[0];
```

Comment:
Why is a the single user a single object in an array? Seems a bit weird and `data[0]` is confusing!